### ISSUE TITLE
latex print error Limit(((1+1/x)**x), x, oo)**2

### ISSUE THREAD

**Issue by satels** [sympy/sympy#11010]
```python
>>> from sympy.abc import x
>>> from sympy import *
>>> latex(Limit(((1+1/x)**x), x, oo)**2)
'\\lim_{x \\to \\infty} \\left(1 + \\frac{1}{x}\\right)^{x}^{2}'
```
the result with latex syntax error:

testing page: https://cdn.mathjax.org/mathjax/latest/test/sample-dynamic.html

But must:
```tex
\left(\lim_{x \to \infty} \left(1 + \frac{1}{x}\right)^{x}\right)^{2}
```

**Comment by Nitin216** [comment]
@satels I am getting right answer. Mine is coming right. I am currently using Python 2.7.11.
The output is:
```
In [2]: latex(Limit(((1+1/x)**x),x,oo)**2)
Out[2]: '\\left(\\lim_{x \\to \\infty} \\left(1 + \\frac{1}{x}\\right)^{x}\\right)^{2}'
```

**Comment by satels** [comment]
@Nitin216 you use last git sympy version?

**Comment by Nitin216** [comment]
@satels Mine is upto date. 
``` python
In [1]: import sympy as s

In [2]: s.__version__
Out[2]: '1.0.1.dev'
```
and also repository is latest too.

**Comment by gxyd** [comment]
@satels this issues was fixed by PR #10824 in sympy git version.
